webui/src/Services/Controls/ControlActionsService.ts

```diff
@@ -9,6 +9,7 @@ export interface IActionEditorService {
 	setValue: (actionId: string, action: ActionInstance | undefined, key: string, val: any) => void
 	performDelete: (actionId: string) => void
 	performDuplicate: (actionId: string) => void
+	setConnection: (actionId: string, connectionId: string | number) => void | undefined
```

Comment:
```suggestion
	setConnection: ((actionId: string, connectionId: string | number) => void) | undefined
```
The `| undefined` is being considered as part of the return value currently, rather than the function being possibly undefined